### PR TITLE
[main < 897-hop-limit-flag] Add hop limit flag

### PR DESCRIPTION
**Pull request by kgolubic**
### Description

Added hop limit flag to the config page.

### Pull request type

Please check what kind of PR this is:

- [ ] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
(paste the link to the issue it closes)

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

**Review by DavIvek — APPROVED** [review]
LGTM!

File: pages/database-management/configuration.mdx [hunk 1]
```diff
@@ -314,6 +314,8 @@ fallback to the value of the command-line argument.
 | log.level                  | Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL.              | no                      |
 | log.to_stderr              | Log messages go to `stderr` in addition to `logfiles`.                                        | no                      |
 | cartesian-product-enabled  | Enforces cartesian product operator during query matching.                                    | no                      |
+| hops_limit_partial_results | Return partial results after the hops limit is reached.                                       | yes                     |
```
Comment (hal-eisen-MG) [inline pages/database-management/configuration.mdx:317]: Explain how the "no" value will change the behavior of the software

File: pages/database-management/configuration.mdx [hunk 2]
```diff
@@ -305,15 +305,17 @@ Memgraph contains settings that can be modified during runtime using a Cypher qu
...
+| enterprise.license         | License key for Memgraph Enterprise.                                                                        | yes                     |
+| server.name                | Bolt server name.                                                                                           | yes                     |
+| query.timeout              | Maximum allowed query execution time. Value of 0 means no limit.                                            | yes                     |
+| log.level                  | Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL.                            | no                      |
+| log.to_stderr              | Log messages go to `stderr` in addition to `logfiles`.                                                      | no                      |
+| cartesian-product-enabled  | Enforces cartesian product operator during query matching.                                                  | no                      |
+| hops_limit_partial_results | If set to `true`, partial results are returned after the hops limit is reached. The default value is `true`.| yes                     |
```
Comment (hal-eisen-MG) [inline pages/database-management/configuration.mdx:317]: What happens when this is `false`?

Comment (DavIvek) [reply pages/database-management/configuration.mdx:317]: Partial results are not returned, instead exception is thrown. Should we add that also?

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Pull request merged**